server/chat.ts

```diff
@@ -1449,46 +1323,36 @@ Chat.CommandContext = CommandContext;
  * or failure (few commands report these) or a Promise for when the command
  * is done executing, if it's not currently done.
  *
- * @param {string} message - the message the user is trying to say
- * @param {Room} room - the room the user is trying to say it in
- * @param {User} user - the user that sent the message
- * @param {Connection} connection - the connection the user sent the message from
+ * @param message - the message the user is trying to say
+ * @param room - the room the user is trying to say it in
+ * @param user - the user that sent the message
+ * @param connection - the connection the user sent the message from
  */
-Chat.parse = function (message, room, user, connection) {
-	Chat.loadPlugins();
-	let context = new CommandContext({message, room, user, connection});
+export function parse(message: string, room: Room, user: User, connection: Connection) {
+	loadPlugins();
+	const context = new CommandContext({message, room, user, connection});
 
 	return context.parse();
-};
-/**
- * @param {string} message
- * @param {User} user
- * @param {User} pmTarget
- * @param {?User} onlyRecipient
- */
-Chat.sendPM = function (message, user, pmTarget, onlyRecipient = null) {
-	let buf = `|pm|${user.getIdentity()}|${pmTarget.getIdentity()}|${message}`;
+}
+export function sendPM(message: string, user: User, pmTarget: User, onlyRecipient: User | null = null) {
+	const buf = `|pm|${user.getIdentity()}|${pmTarget.getIdentity()}|${message}`;
 	if (onlyRecipient) return onlyRecipient.send(buf);
 	user.send(buf);
 	if (pmTarget !== user) pmTarget.send(buf);
 	pmTarget.lastPM = user.userid;
 	user.lastPM = pmTarget.userid;
-};
+}
 
-Chat.package = {};
+export let packageData = {};
```

Comment:
Wait, why was this renamed?

Reply:
`package` is a restricted word, so you can't call a variable that.

Reply:
Fair.